### ISSUE TITLE
bonus mutiplier?

### ISSUE THREAD

**Issue by NapsterInBlue** [pvpoke/pvpoke#126]
Excuse my ignorance, but I was noticing that the battler was showing higher fast move damage than I had expected. Particularly in [this match-up](https://pvpoke.com/battle/1500/charmander/bulbasaur/11/0-3-4/1-4-1/), where I expected the Embers to be hitting for 7, not 8.

I worked it out a few times by hand and couldn't get to 8, so I took a peek at the codebase. When I did, I found consistent use of `bonusMultiplier = 1.3`, particularly [in the main damage function](https://github.com/pvpoke/pvpoke/blob/fbae9b7fa272ba9079a2e3f736cb7cacd93de7c9/src/js/battle/Battle.js#L204).

I've been looking over various wikis ([1](https://pokemonbuddies.com/pokemon-go-pvp-guide-comprehensive-move-damage-explanation/), [2](https://gamepress.gg/pokemongo/damage-mechanics), [3](https://pokemongohub.net/post/questions-and-answers/move-damage-output-actually-calculated/)) on damage calculation, and the only thing I omitted from my calculations was the "friendship bonus." But even then, that'd give a 10% boost, tops, to that `floor(...)` term.

So where does this "times 130%" come from?

**Comment by pvpoke** [comment]
Hey, thanks for the question! PvP has a global x1.3 multiplier to all damage.  In the [game master file](https://github.com/PokeMiners/game_masters)'s `"COMBAT_SETTINGS"` template ("combat" refers to PvP), this corresponds to the `"fastAttackBonusMultiplier"` and `"chargeAttackBonusMultiplier"` properties which have the value of 1.3.

**Comment by NapsterInBlue** [comment]
I N T E R E S T I N G

First off, I appreciate you taking the time to reply!

I'll admit, this threw me for a loop for longer than I care to admit. Documenting in case someone else comes along wondering the same thing.

Inspecting the .json, I got it in my head that `templateId: COMBAT_SETTINGS` was used for *both* PvP and PvE-- particularly because I couldn't grok what `roundDurationSeconds` meant in the context of PvP (still don't, tbh)

![image](https://user-images.githubusercontent.com/3505365/97888932-b724aa80-1cf9-11eb-9ddf-726dd2c40ffb.png)

... but then I saw that the field is also used in `templateId: BATTLE_SETTINGS` , where we also see expected PvP fields, such as `maximumAttackersPerBattle`, `dodgeDurationMs`, `dodgeDamageReductionPercent`, etc

![image](https://user-images.githubusercontent.com/3505365/97889098-ee935700-1cf9-11eb-90a6-9b46454a888c.png)


But to ask the blatant question in summary: *you're of the opinion that the resources you'd find googling "pogo damage calculation" are all missing this critical multiplier?* Because I think I've sufficiently convinced myself to the point where I'm inclined to agree.
